pkg/services/grn/grn.go

```diff
@@ -0,0 +1,61 @@
+package grn
+
+import (
+	"fmt"
+	"strings"
+)
+
+type GRN struct {
+	// Service identifies the service responsible for the resource.
+	Service Service
+
+	// TenantID is specific to hosted grafana and will be omitted in other
+	// environments.
+	TenantID string
```

Comment:
I think we need TenantID *OR* OrgID, not both

Reply:
🤔 Are tenant and orgIDs the same field in the code base? These were separate in the design doc as well.
I'm concerned about the ambiguity - if I have a GRN, how would I know if the (tenantID|orgID) field contained an org or tenant id? - but I also don't know much about how our enterprise and hosted offerings differ from OSS (in code).

Reply:
I'm leaving both in for the time being at @zserge 's suggestion - we can refactor this whenever org and tenantID are merged into one somethingID.

Reply:
orgId exists in grafana core, tenantid exists in cloud -- we need to flatten this into one thing so that we do not add confusion.  They will *NEVER* exist at the same time and we want to get that established ASAP.

We should also decide if it should be a string or number

Reply:
> we can refactor this whenever org and tenantID

Given this format, how do we refactor?